Мне все-таки не удалось выложить бота на heroku (по крайней мере, по состоянию на 15:35). Мне выдается ошибка R14 (memory quota exceeded), с которой я, кажется, сделать ничего не могу. Сообщение о том, что приложение упало, не появляется довольно долго (потому что heroku пытается перезапустить его, но объем используемой памяти во время этого процесса только увеличвается), но потом оно все же падает. А на pythonanywhere мне не даст выложиться запрос на скачивание со сторонних сайтов. Вот так. Но если запускать вручную, все работает, эх.

В таком случае код, наверное, будет удобнее смотреть в .ipynb, но папку с фалйами .py я тоже добавила.

Ссылка на файлы с текстами: [гугл-диск](https://drive.google.com/open?id=1kGWmYGrHQ8SNCoOcAZASeUVMhWvGuBRC).

![скриншот1](https://github.com/maryoocean/Programming-2/blob/master/Homeworks/final%20project/screeshots/%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA%20%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0%202019-06-19%20%D0%B2%2014.59.51.png?raw=true "скриншот1")
![скриншот2](https://github.com/maryoocean/Programming-2/blob/master/Homeworks/final%20project/screeshots/%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA%20%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0%202019-06-19%20%D0%B2%2015.00.08.png?raw=true "скриншот2")
![скриншот3](https://github.com/maryoocean/Programming-2/blob/master/Homeworks/final%20project/screeshots/%D0%A1%D0%BD%D0%B8%D0%BC%D0%BE%D0%BA%20%D1%8D%D0%BA%D1%80%D0%B0%D0%BD%D0%B0%202019-06-19%20%D0%B2%2015.18.01.png?raw=true "скриншот3")
